backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -316,6 +317,15 @@ class ReviewResponse(models.Model):
         help_text=_("The response of the author of the destruction list to a review."),
     )
     created = models.DateTimeField(auto_now_add=True)
+    processing_status = models.CharField(
```

Comment:
I think processing status here should be computed based on the items status if possible.